chrome/content/coniks/context.js

```diff
@@ -0,0 +1,73 @@
+this.EXPORTED_SYMBOLS = ["AccountContext", "ConiksContext", "ContextsFile"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("chrome://otr/content/helpers.js");
+
+var OS;
+
+class ConiksContext {
+  constructor(opts) {
+    opts = opts || {};
+
+    if (opts.account === null) {
+      return null;
+    }
+    this.account = opts.account;
+    // user policies
+    this.allowUnsignedKeyChange = opts.allowUnsignedKeyChange === null || opts.allowUnsignedKeyChange === undefined ?
```

Comment:
Would it be nicer if these were called, `signedKeyChange` and `privateLookups` so that all these falsy values are synonymous?  (Presenting the option to the user can remain as you have it though.)

Also, the ternary operator is unnecessary here.

```
this.allowUnsignedKeyChange =
  opts.allowUnsignedKeyChange === null ||
  opts.allowUnsignedKeyChange === undefined ||
  opts.allowUnsignedKeyChange;
```